I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 278, in script
    python:
ScriptError: could not find label 'test1'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 660, in main
    run(restart)
  File "renpy/main.py", line 146, in run
    renpy.execution.run_context(True)
  File "renpy/execution.py", line 938, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 278, in script
    python:
  File "renpy/script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'test1'.

Windows-10-10.0.19041 AMD64
Ren'Py 7.5.3.22090809
候鸟 准正式版 0.99
Mon Jan  9 19:19:18 2023
